pkg/storage/filesystem_test.go

```diff
@@ -0,0 +1,32 @@
+package storage
+
+import (
+	"testing"
+)
+
+func TestGetFileSystems(t *testing.T) {
+	fstypes, err :=  internalGetFilesystems("tests/filesystems")
+	expected := [3]string{"ext4", "ext3", "vfat"}
+	if err != nil{
+		t.Error("Failed to open the proc filesystems")
+	}
+	if len(fstypes) != len(expected){
+		t.Errorf("Expected '%v', but resulted with '%v'", expected, fstypes)
+	}
+	for index, value := range fstypes{
+		if value != expected[index]{
+			t.Error("Wrong filesystems appened.")
```

Comment:
Can make this more informative by printing the expected vs. actual like you did above

Reply:
Would it be more helpful if I specifically printed which value it failed at, or should I print out the entire fstypes slice and expected slice?

Reply:
Yeah the specific value would be good!